Lib/ftplib.py

```diff
@@ -102,7 +102,9 @@ class FTP:
     sock = None
     file = None
     welcome = None
-    passiveserver = 1
+    passiveserver = True
+    # Disables https://bugs.python.org/issue43285 security if set to True.
+    use_untrusted_server_pasv_ipv4_addr = False
```

Comment:
I suggest the name "trust_server_pasv_ipv4_addr".

Reply:
Done.  and I mention it in the NEWS entry.